documentation/install_from_packages.markdown

```diff
@@ -95,6 +81,4 @@ By default, Puppet Server is configured to use 2GB of RAM. However, if you want
 
 2. Restart the `puppetserver` service after making any changes to this file.
 
-## Reporting Issues
-
-Submit issues to our [bug tracker](https://tickets.puppet.com/browse/SERVER).
+> Note: Submit issues to our [bug tracker](https://tickets.puppet.com/browse/SERVER).
```

Comment:
I feel like putting this in a Note sort of hides this info.. It also seems like it's specific to the `Running Puppet Server on a VM`? which it's not.

I kinda like the separate `Reporting Issues` section. Is there a reason we removed it?

Reply:
Yes you're right it does look like it belongs to that section, so I have moved it to the top now. Let me know what you think.

I initially removed it to keep this page consistent with the Puppet docs pages, and also other Server docs pages, which don't have a report an issue section.  Is it more likely that they would need to report a problem when installing? If so we can put it back, or if you think the note at the top works just as well, we can leave it there.

Reply:
ahh, I'm cool with removing the section in order to be consistent 👍